src/sagemaker_containers/_env.py

```diff
@@ -84,6 +137,31 @@
 resource_config_file_dir = os.path.join(input_config_dir, RESOURCE_CONFIG_FILE)  # type: str
 
 
+def _create_training_directories():
+    """Creates the directory structure and files necessary for training under the base path
+    """
+    logger.info('Creating a new training folder under %s .' % base_dir)
+
+    os.makedirs(model_dir)
+    os.makedirs(input_config_dir)
+    os.makedirs(output_data_dir)
+
+    _write_json({}, hyperparameters_file_dir)
+    _write_json({}, input_data_config_file_dir)
+
+    host_name = socket.gethostname()
```

Comment:
Will this work for every docker container?

Reply:
I believe so. It works for Windows, Mac, and Unix https://docs.python.org/2/library/socket.html